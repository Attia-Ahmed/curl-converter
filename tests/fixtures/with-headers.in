curl -H "X-Header-One: 1" -H 'X-Header-Two: 2' https://example.com
