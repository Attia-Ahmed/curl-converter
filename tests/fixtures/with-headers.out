Http::withHeaders([
        'X-Header-One' => 1,
        'X-Header-Two' => 2,
    ])
    ->get('https://example.com');
